### PR TITLE
Add Python 3.6 and Drop Python 3.3

### PR DESCRIPTION
**Pull request by jamadden**
- Badges
- Enable coveralls and add a coverage environment
  - 100% coverage
- Rename a .txt to .rst.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,35 +47,40 @@ def read(*rnames):
...
+          'zope.component',
+          'zope.event',
+          'zope.interface',
+          'zope.location',
+          'zope.publisher',
+          'zope.schema',
+          'zope.tales',
+      ],
+      include_package_data=True,
+      tests_require=['zope.testing', 'zope.browserpage'],
```
Comment (icemac) [inline setup.py]: Is there a reason not to deduplicate the test dependencies?

File: tox.ini [hunk 2]
```diff
@@ -1,20 +1,21 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3
+    py27,py34,py35,py36,pypy,pypy3
```
Comment (icemac) [inline tox.ini]: Same here about the `coverage` environment as in zopefoundation/zope.site#7.

**Review by icemac — APPROVED** [review]
Thank you and LGTM.

**Comment by icemac** [comment]
BTM I made you an owner of this package on PyPI.

File: setup.py (hunk 1 above)
Comment (jamadden) [reply setup.py]: Nope, just lazy I guess. Done.

File: tox.ini (hunk 2 above)
Comment (jamadden) [reply tox.ini]: Done.

**Comment by jamadden** [comment]
Thanks. I've released 4.1.0 to PyPI.

**Pull request merged**